src/components/edit-movie-form/edit-movie-form.tsx

```diff
@@ -0,0 +1,112 @@
+import { Button } from 'components/button';
+import { Input } from 'components/input';
+import { ReleaseDatePicker } from 'components/release-date-picker';
+import { Select } from 'components/select';
+import React, { FC, FormEvent, useState } from 'react';
+import {
+  CloseButton,
+  EditMovieButtonContainer,
+  EditMovieFormContainer,
+  EditMovieFormInner,
+  EditMovieFormTitle,
+  EditMovieFormWrapper,
+} from './styles';
+
+const initialValues = {
+  genre: '',
+  movie: '',
+  overview: '',
+  rating: '',
+  runtime: '',
+  title: '',
+};
+
+const selectOptions = [
+  { id: 1, name: 'Crime' },
+  { id: 2, name: 'Documentary' },
+  { id: 3, name: 'Horror' },
+  { id: 4, name: 'Comedy' },
+];
+
```

Comment:
You have code duplication or at least very similar code as in add-movie-form. It will be nice to utilize this functionality into a separate component or create reusable customer hook